jpa/eclipselink/src/main/java/example/springdata/jpa/eclipselink/domain/Race.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package example.springdata.jpa.eclipselink.domain;
+
+
+/**
+ * Entity showing use of {@link org.eclipse.persistence.annotations.UuidGenerator} from
+ * the EclipseLink JPA Provider. Does not extend the AbstractPersistable because of the UUID generator
+ *
+ * @author Jeremy Rickard
+ */
+
+import org.eclipse.persistence.annotations.UuidGenerator;
+
+import javax.persistence.*;
+import java.util.Date;
+import java.util.List;
+
+@Entity
+public class Race {
```

Comment:
Let's try to avoid as much customizations as possible as the focus of the example is clearly on the `EclipseLink` setup, not more advanced JPA mappings.